Resources/bin/build_demo.sh

```diff
@@ -41,7 +41,7 @@ composer install --prefer-dist --no-interaction --ignore-platform-reqs --no-plug
 cd /tmp/Symfony
 rm -f UPGRADE*
 mv .gitignore keep.gitignore
-rm -rf app/cache/* app/logs/* .git*
+rm -rf app/cache/* app/logs/* .git/
```

Comment:
this is not needed. the gitignore is kept above, and this would not remove ``.gitattributes`` anymore